packages/webdoc-default-template/tmpl/document.tmpl

```diff
@@ -1,78 +1,60 @@
 <?js
-    /*::
-    import type {Doc} from "@webdoc/model";
-
-    type DocumentData = {
-      docs: Doc[]
-    }
-     */
-
-    const docData = obj;
-    const docList = obj.docs;
-
-    for (let i = 0, j = docList.length; i < j; i++) {
-      const doc = docList[i];
-  ?>
-    <h1 class="document-title"><?js= doc.path ?></h1>
-    <hr />
-
-    <?js= this.plugins.signature.generateSignature(doc, "signature") ?>
-
-    <hr class="secondary-separator" />
-    <?js= doc.brief ?>
-    <?js= doc.description ?>
-
-    <?js if (doc.type === "ClassDoc") {
-        /* Constructor */
-        const ctorDoc = this.getConstructor(doc);
-
-        if (ctorDoc) {
-      ?>
-        <h2 class="section-title">Constructor</h2>
-        <hr />
+const
+  docData = obj,
+  docList = obj.docs;
+
+docList.forEach((doc) => {
+?>
+<div class="document">
+  <h1 class="document__title"><?js= doc.path ?></h1>
+  <?js= this.partial("components/signature.tmpl", doc) ?>
+
+  <div class="document__brief"><?js= doc.brief ?></div>
+  <div class="ducument__desctiption"><?js= doc.description ?></div>
```

Comment:
This should be `document__description` (spelling error). 😅

Reply:
Oh, yes. I'll fix this up.

Reply:
Fixed in 0f16fbc8945b0f43fd00211197439447a8a4dfdc.